### ISSUE TITLE
[Bug]: Can't select windows versions before XP?

### ISSUE THREAD

**Issue by SeanRamey** [bottlesdevs/Bottles#1446]
### Describe the bug

I created a 32bit bottle, yet, I cant change the Windows version to anything before XP. I'd like to select Windows 98.

### To Reproduce

1. Create 32 bit custom bottle
2. Open bottle
3. Try to select Windows 98 from the Windows Version option
4. Observe that you can't.

### Package

AUR ("bottles" package)

### Distribution

EndeavorOS

### Health Check

```shell
Display:
    X.org: true
    X.org (port): :0
    Wayland: false
Graphics:
    vendors:
        nvidia:
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/share/vulkan/icd.d/nvidia_icd.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.17.5-arch1-1
Distro:
    Name: EndeavourOS
    Version: Unknown
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.35'
Bottles_envs: null
```

### Display Protocol

X11

### Display Backend used by Bottles

X11

**Comment by pietrodicaprio** [comment]
Already managed with #1421 , Win98 and Win95 are going to be available from next release (2022.5.14)

**Comment by DJCrashdummy** [comment]
<del>interesting... i'm using Bottles 51.13 and still the oldest version i can select (within Bottles) is Windows XP. :confused:</del>
revisiting this issue: i guess i mistakenly used a 64bit bottle and assumed it was 32bit. 🤷 ...at least Win95 and Win98 are available options for 32bit bottles.

it seems that #1526 is currently the only open issue with a little bit more information regarding the topic older versions.
